gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/sql/execute/SnappyUpdateDeleteResultSet.java

```diff
@@ -40,7 +40,7 @@ public int modifiedRowCount() {
       rowCount = 0;
       try {
         ExecRow row;
-        while ((row = getNextRow()) != null) {
+        while ((row = getNextRow()) != null && row.nColumns() > 0) {
```

Comment:
Here row.nColumns should always be 1. Right? If yes then should we just do '== 1' instead or throw exception if nColumns is anything except 1.